src/classes/CRLP_Batch_Base_NonSkew.cls

```diff
@@ -120,10 +120,9 @@ public abstract class CRLP_Batch_Base_NonSkew extends CRLP_Batch_Base {
                 // Only use the related records filter if not disabled in settings and not running unit tests
                 if (customizableRollupSettings.Disable_Related_Records_Filter__c == false && !Test.isRunningTest()) {
```

Comment:
@force2b Michael, can you provide any info as to why we are skipping the block below when in Test context?